support-workers/src/test/scala/com/gu/support/workers/integration/DigitalSubscriptionGiftRedemptionIntegrationSpec.scala

```diff
@@ -19,47 +19,48 @@ import scala.concurrent.Future
 
 @IntegrationTest
 class DigitalSubscriptionGiftRedemptionIntegrationSpec extends AsyncLambdaSpec with MockContext {
-  val createZuoraHelper = new CreateZuoraSubscriptionHelper()
-
-  val createSubRequestId = UUID.randomUUID()
-  val redeemSubRequestId = UUID.randomUUID()
-  val giftCode = new GiftCodeGeneratorService().generateCode(Annual)
-  SafeLogger.info(s"Gift code = ${giftCode}")
-  val mockCodeGenerator = mock[GiftCodeGeneratorService]
-  when(mockCodeGenerator.generateCode(any[BillingPeriod])).thenReturn(giftCode)
```

Comment:
all this was being run when the test run initialised so the log statements were appearing at the top of the logs.